core/src/test/java/com/exadel/etoolbox/linkinspector/core/servlets/ReplaceByPatternServletTest.java

```diff
@@ -130,9 +124,10 @@ void setup() {
 
         context.registerInjectActivateService(
                 new MockHttpClientBuilderFactory(),
-                ImmutableMap.of(
-                        "statusCode", org.apache.http.HttpStatus.SC_NOT_FOUND,
-                        "statusMessage", "Not Found")
+                Collections.unmodifiableMap(new HashMap<String, Object>() {{
```

Comment:
See the comment on the unmodifiable map above

Reply:
done